test/regress/regress1/strings/str-code-sat.smt2

```diff
@@ -0,0 +1,24 @@
+(set-logic QF_S)
```

Comment:
Nit: shouldn't this be `QF_SLIA` since there are integer additions/comparisons?

Reply:
Yes, good catch.